mpa/modules/models/classifiers/sam_classifier.py

```diff
@@ -30,6 +30,8 @@ class SAMImageClassifier(ImageClassifier):
     """SAM-enabled ImageClassifier"""
 
     def __init__(self, task_adapt=None, **kwargs):
+        if 'multilabel' in kwargs:
```

Comment:
@harimkang this multilabel flag is needed only for state_dict_hook. How do you think about separating state_dict_hook into two cases: 1) backbone, 2) head
If we separate this, each classifier will not have to remove multilabel argument. But I think state_dict hook in heads might be implemented.